docker-compose.yml

```diff
@@ -1,11 +1,11 @@
 version: '3.1'
 services:
 
-  redis-bapv:
+  redis:
```

Comment:
Accidental change but makes more sense when you're trying to run other projects that use redis at the same time and use `redis` as the service name.